src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String REPORT_TITLE = "fruit,quantity\n";
+    private static final String SEPARATOR = ",";
+    private static Map<String, Integer> TOTAL_FRUIT_MAP;
```

Comment:
it's not constant, even if it was final - map is mutable, you can put new key-value pairs, delete existing ones, rewite etc. So don't use constant case